doc/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	add_executable (markdownlinkconverter markdownlinkconverter/markdownlinkconverter.c)
+	find_util (markdownlinkconverter MARKDOWN_LINK_CONVERTER "")
+
```

Comment:
There are two output parameters, you should request both and concatinate them to MARKDOWN_LINK_CONVERTER.

I added some docu for find_util: 9ee141a06df3ab3e8e0933f22c72c498f5483557

Can you give arguments to the doxygen filter? If not, it won't work with mingw.